mars-cli/mars_lib/submit.py

```diff
@@ -101,7 +114,7 @@ def submission(
         # Submit to Biosamples
         biosamples_result = submit_to_biosamples(
             isa_json=isa_json,
-            biosamples_credentials=user_credentials,
+            biosamples_credentials=user_credentials[TargetRepository.BIOSAMPLES.value],
```

Comment:
Should this be `AuthProvider.WEBIN.value` instead? (And same for the ENA usages below)
Not sure if we need some kind of `TargetRepository -> AuthProvider` mapping, or whether just defining it implicitly in this method is enough.

Reply:
it should, I was just testing it out and got following error:

```
    submission(
  File "/home/bedro/.pyenv/versions/3.11.3/envs/mars/lib/python3.11/site-packages/mars_lib/submit.py", line 117, in submission
    biosamples_credentials=user_credentials[TargetRepository.BIOSAMPLES.value],
                           ~~~~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```